bin/npg_move_runfolder

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env perl
+#########
+# Author:        js10
+# Created:       2018-05-30
+#
+
+use strict;
+use warnings;
+
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Carp;
+use autodie qw(:all);
+
+use Monitor::Staging;
+use Monitor::RunFolder::Staging;
+
+our $VERSION = '0';
+
+use Readonly;
+
+my $mon = Monitor::RunFolder::Staging->new_with_options(status_update => 0);
+my @ms = $mon->move_to_analysis();
```

Comment:
It is still possible to overwrite status_update from the command line. Do we want this?

Reply:
I'm happy to keep that "free" functionality.